stable/coredns/README.md

```diff
@@ -5,7 +5,8 @@
 # TL;DR;
 
 ```console
-$ helm install --name coredns --namespace=kube-system stable/coredns
+$ helm repo add coredns https://lablabs.github.io/deployment
+$ helm --namespace=kube-system install coredns coredns/coredns
```

Comment:
Adding services to kube-systems has it's own problems(cluster-autoscaler). Maybe we don't have to suggest that to our users.